src/components/featureItem.tsx

```diff
@@ -0,0 +1,20 @@
+interface Props {
+  children: string;
+  title: string;
+  src: string;
+  order?: string;
+}
+
+export const FeatureItem = ({ children, title, src, order }: Props) => {
+  return (
+    <div
+      className={`flex ${order} justify-around items-center mb-20 text-xl font-medium text-yellow-900`}
+    >
+      <img className="w-1/3 rounded-4xl" src={src} alt="Barber works" />
+      <div className="w-1/3">
```

Comment:
There are two blocks, but each takes 33.3% of width. Consider using `w-1/2` for one of them